server/middleware/users.coffee

```diff
@@ -26,6 +26,9 @@ config = require '../../server_config'
 utils = require '../lib/utils'
 CLASubmission = require '../models/CLASubmission'
 Prepaid = require '../models/Prepaid'
+middleware = {
+  subscriptions: require './subscriptions'
+}
```

Comment:
Thinking again, having a subset of middleware could be confusing, `middleware` should be the same object throughout the server. Would rather either subscriptions just get required here instead of middleware, OR require middleware from within the delete function. Or, even better, pull the function out of subscriptions entirely, which would get required by both middleware files.